tcms/tests/factories.py

```diff
@@ -260,7 +260,7 @@ class Meta:
     summary = factory.LazyAttribute(lambda obj: 'Summary of bug %s' % obj.bug_id)
     description = ''
     bug_system = factory.LazyFunction(lambda: BugSystem.objects.all()[0:1][0])
```

Comment:
While at it, update the query here to be `BugSystem.objects.first()` because this is what the indexing does anyway.

Reply:
It throws a `TypeError: 'BugSystem' object is not subscriptable`.